analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.cs

```diff
@@ -172,5 +172,83 @@ public void Execute_PersistSymbols_InsideBlock()
             static ISymbol LocalReferenceOperationSymbol(IOperationWrapperSonar operation) =>
                 ((ILocalReferenceOperation)operation.Instance).Local;
         }
+
+        [TestMethod]
+        public void ExitReached_SimpleFlow()
+        {
+            var exitCount = 0;
+            SETestContext.CreateCS("var a = true;", new ExitReachedTestCheck(state => CountExits(state, ref exitCount)));
+            Assert.AreEqual(1, exitCount);
```

Comment:
Should shoudly.. for purpose of the `ValidateExitReachCount` internals
```suggestion
            exitCount.Should().Be(1);
```